src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -231,9 +245,16 @@ function MoneyRequestAmountForm({amount, currency, isEditing, forwardedRef, onCu
                     onKeyPress={textInputKeyPress}
                 />
             </View>
+            {!_.isEmpty(formError) && firstPress && (
+                <FormHelpMessage
+                    style={[styles.ph5]}
+                    isError
+                    message={translate(formError)}
+                />
+            )}
```

Comment:
I'd say https://github.com/Expensify/App/issues/29204 is a regression as it could have been handled as part of this PR.